modules/docs/llm/upgrade-guides/16.0-UPGRADE-GUIDE.md

```
@@ -556,9 +645,10 @@ our team. We'd be very happy to walk you through the process to set you up for s
 
 ### Instructions
 
-The easiest way to run our codemod is to use `npx` in your terminal.```sh
-npx @workday/canvas-kit-codemod v${canvasKitMajorVersionNumber} [path]
-```
+The easiest way to run our codemod is to use `npx` in your terminal.```sh npx
+@workday/canvas-kit-codemod v${canvasKitMajorVersionNumber} [path]
+
+````
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)